streams/src/test/java/org/apache/kafka/streams/perf/SimpleBenchmark.java

```diff
@@ -469,13 +469,18 @@ private void processStreamWithWindowStore(final String topic) {
         setStreamProperties("simple-benchmark-streams-with-store");
 
         final StreamsBuilder builder = new StreamsBuilder();
-        final StoreBuilder<WindowStore<Integer, byte[]>> storeBuilder
-                = Stores.windowStoreBuilder(Stores.persistentWindowStore("store",
+
+        final StoreBuilder<WindowStore<Integer, byte[]>> storeBuilder = Stores.windowStoreBuilder(
+            Stores.persistentWindowStore(
+                "store",
                 AGGREGATE_WINDOW_SIZE * 3,
-                3,
                 AGGREGATE_WINDOW_SIZE,
-                false),
-                INTEGER_SERDE, BYTE_SERDE);
+                false,
+                60_000L
```

Comment:
This is the arithmetic solution of the previous formula `max((retention / (numSegments - 1)), 60,000) = max((1000 * 3 / (3 - 1)), 60,000) = max(3,000/2, 60,000) = 60,000`. I point this out because it's important not to change any parameters on the benchmark.